Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/CiscoEmailSecurityApplianceIronPortV2.yml

```diff
@@ -0,0 +1,1186 @@
+        When view_by = sender: sender_addresses and recipient_list are mandatory.
+      required: false
+      isArray: false
+      defaultValue: recipient
+      auto: PREDEFINED
+      predefined:
+      - recipient
+      - sender
+    - name: recipient_list
+      description: A comma-separated list of recipients to append.
+      required: false
+      isArray: true
+    - name: sender_list
+      description: A comma-separated list of senders to append.
+      required: false
+      isArray: false
+    - name: recipient_addresses
+      description: A comma-separated list of recipients to append.
+      required: false
+      isArray: false
+    - name: sender_addresses
+      description: A comma-separated list of senders to append.
+      required: false
+      isArray: false
+    outputs: []
+  - name: cisco-esa-list-entry-edit
+    description: Edit spam quarantine blocklist/safelist entry. Using this command will overide the existing value.
+    arguments:
+    - name: entry_type
+      description: List entry type.
+      required: true
+      isArray: false
+      auto: PREDEFINED
+      predefined:
+      - blocklist
+      - safelist
+    - name: view_by
+      description: |-
+        Edit list entry by recipient/sender.
+        When view_by = recipient: recipient_addresses and sender_list are mandatory.
+        When view_by = sender: sender_addresses and recipient_list are mandatory.
+      required: false
+      isArray: false
+      defaultValue: recipient
+      auto: PREDEFINED
+      predefined:
+      - recipient
+      - sender
+    - name: recipient_list
+      description: A comma-separated list of recipients to edit.
+      required: false
+      isArray: true
+    - name: sender_list
+      description: A comma-separated list of senders to edit.
+      required: false
+      isArray: false
+    - name: recipient_addresses
+      description: A comma-separated list of recipients to edit.
+      required: false
+      isArray: false
+    - name: sender_addresses
+      description: A comma-separated list of senders to edit.
+      required: false
+      isArray: false
+    outputs: []
+  - name: cisco-esa-list-entry-delete
+    description: Delete spam quarantine blocklist/safelist entry.
+    arguments:
+    - name: entry_type
+      description: List entry type.
+      required: true
+      isArray: false
+      auto: PREDEFINED
+      predefined:
+      - blocklist
+      - safelist
+    - name: view_by
+      description: |-
+        Delete list entry by recipient/sender.
+        When view_by = recipient: recipient_list is mandatory.
+        When view_by = sender: sender_list is mandatory.
+      required: false
+      isArray: false
+      defaultValue: recipient
+      auto: PREDEFINED
+      predefined:
+      - recipient
+      - sender
+    - name: recipient_list
+      description: List of recipient/sender addresses to delete.
```

Comment:
why is this recipient/sender?  The argument is for recipient_list.  Shouldn't it just be recipient?  And is it just the addresses that are deleted or the recipient in general?

Reply:
This argument implementation is according to the API usage, and was discussed with Meital.